components/vf-banner/vf-banner.scss

```
@@ -0,0 +1,82 @@
+$vf-data-protection-banner-color--background: set-color(vf-color-gray-dark);
+$vf-color--text: set-color(vf-color-white);
+$vf-data-protection-banner-color--link: inherit;
+
+$vf-data-protection-banner-padding: map-get($vf-spacing-map, vf-spacing-r);
+
+.vf-banner {
+  box-sizing: border-box;
+  padding: 0 12px;
+
+  .vf-tag {
+    margin-right: 16px;
+  }
+}
+
+.vf-banner--fixed {
+  display: grid;
+  left: 0;
+  grid-column: 1 / -1;
+  grid-template-columns:
+    [full-start]
+      minmax(1em, auto)
+      [main-start]
+        minmax(18em, 81.25em) // 1300px
+      [main-end]
+      minmax(1em, auto)
+    [full-end];
+  padding: 0;
+  position: fixed;
+  width: 100%;
+  z-index: 5150;
+}
+
+.vf-banner__content {
+  align-items: center;
+  align-content: center;
+
+  @media (max-width: 63.9375em) {
+    display: block; // overideing the .emlb-grid code
+  }
+
+  @media (min-width: $vf-breakpoint-l) {
+    grid-column: main;
+  }
+}
+
+.vf-banner--top {
+  top: 0;
+}
+
+.vf-banner--bottom {
+  bottom: 0;
+}
+
+.vf-banner--phase {
+  background-color: #FFFADC;
+}
+
+.vf-banner--gdpr {
+  background-color: set-color(vf-color-gray-dark);
```

Did you mean `$vf-data-protection-banner-color--background`?
```suggestion
  background-color: $vf-data-protection-banner-color--background;
```

---

well - I will do ;) 

need to think about how best to 'theme' them really -- do we have specific use cases or do we have generic light / dark versions? w/ light being the yellow and dark being the dark grey?

---

... to me, I think the specific use cases should be EMBL patterns (embl-banner) and more generic use cases for the vf-core "vf-banner--light"